controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -49,6 +52,22 @@ static Voltage MPXV5004_PressureToVoltage(Pressure pressure) {
   return volts(3.3f * (0.2f * pressure.kPa() + 0.2f));
 }
 
+// Function that helps change the readings by setting the pressure sensor pins,
+// advancing time and then getting sensors readings
+static SensorReadings update_readings(Duration dt, Pressure patient_pressure,
+                                      Pressure inflow_pressure,
+                                      Pressure outflow_pressure,
+                                      Sensors sensors) {
```

Comment:
This passes "sensors" by value, that can't be right, because that means it doesn't carry over their state. Please pass by pointer.

Reply:
Done